addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/PopupMenuEvidence.java

```diff
@@ -75,6 +75,23 @@ public boolean isEnableForComponent(Component invoker) {
                         addMenuItem(p, ExtensionSearch.Type.Response);
                     }
                 }
+                for (Map<String, Map<String, Map<String, AppPattern>>> map : app.getDom()) {
+                    for (Map.Entry<String, Map<String, Map<String, AppPattern>>> ap :
+                            map.entrySet()) {
+                        for (Map.Entry<String, Map<String, AppPattern>> p :
+                                ap.getValue().entrySet()) {
+                            for (Map.Entry<String, AppPattern> pa : p.getValue().entrySet()) {
+                                Pattern pat =
+                                        Pattern.compile(
+                                                pa.getKey()
+                                                        + ".*"
+                                                        + pa.getValue().getJavaPattern().pattern());
+                                addMenuItem(pat, ExtensionSearch.Type.Response);
+                            }
+                        }
+                    }
+                }
+
```

Comment:
I'm not sure this is going to work properly, I suspect that too many messages will be matched because the assembled property pattern won't match as specifically as the whole DOM selector.

Note I haven't tested IRL, that's just based on reading.